kubernetes/templates/deployment.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ppa-predictions
+  namespace: ppa
+  labels:
+    deployment: ppa-predictions
+spec:
+  selector:
+    matchLabels:
+      app: ppa-predictions
+  template:
+    metadata:
+      annotations:
+        seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
+        apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
+        seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
+        apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
+      labels:
+        app: ppa-predictions
+    spec:
+      priorityClassName: "normal-priority"
+      securityContext:
+        runAsUser: 1000  # prevent running as root
+        fsGroup: 1000
+      containers:
+        - name: ppa-predictions
+          image: "nodeenergy.azurecr.io/ppa-predictions:latest"
+          command: ["uvicorn", "src.main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8001"]
```

```suggestion
          command: ["uvicorn", "src.main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8001", "--workers", "2"]
```

Even when running on 1 core, should use 2 workers; Otherwise, because it's running on asyncio, CPU bound things will block the main thread, therefore the event loop, and accordingly prevent concurrent requests from running (efficiently).